core/cloudflow-operator/src/main/scala/cloudflow/operator/runner/FlinkRunner.scala

```diff
@@ -112,7 +112,7 @@ object FlinkRunner extends Runner[CR] {
           name = name,
           namespace = namespace,
           annotations = Map("prometheus.io/scrape" -> "true", "prometheus.io/port" -> PrometheusConfig.PrometheusJmxExporterPort.toString),
-          labels = labels,
+          labels = labels ++ getLabels(podsConfig, PodsConfig.CloudflowPodName),
```

Comment:
I don't think this works correctly when they specify labels in the specific pods for job-manager and task-manager? The same very likely for driver and executor https://cloudflow.io/docs/current/develop/cloudflow-configuration.html#_kubernetes_container_settings

Reply:
Done